eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use Doctrine\DBAL\Connection;
+use Exception;
+use eZ\Bundle\EzPublishCoreBundle\ApiLoader\RepositoryConfigurationProvider;
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use PDO;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\Output;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CleanupVersionsCommand extends Command
+{
+    const DEFAULT_REPOSITORY_USER = 'admin';
+
+    const VERSION_DRAFT = 'draft';
+    const VERSION_ARCHIVED = 'archived';
+    const VERSION_PUBLISHED = 'published';
+    const VERSION_ALL = 'all';
```

Comment:
The mapping logic could have been done like so:
```php
const VERSION = [
    'draft' => VersionInfo::STATUS_DRAFT,
    'archived' => VersionInfo::STATUS_ARCHIVED,
    'published' => VersionInfo::STATUS_PUBLISHED,
];
```
But "all" is something else here.

Reply:
@andrerom Improved mapping a little bit in https://github.com/ezsystems/ezpublish-kernel/pull/2431/commits/d75927b1613024dcddbee08cfa27643f2ed06cf1, so now, there is only one place with usage of `VersionInfo`.